src/views/GameListView.jsx

```diff
@@ -132,6 +143,15 @@ const GameListView = () => {
       <Content>
         <Filter>
           <h2 className="incentive">{t("Start a game now")}</h2>
+          <input
```

Comment:
You can use a real form with react-final-form here to prepare further filtering features.

Reply:
Later!